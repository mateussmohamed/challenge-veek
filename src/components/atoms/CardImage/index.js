import styled from 'styled-components'

const CardImage = styled.img`
  width: 100%;
  height: 170px;
  background-color: #25262c;
  background-image: url('/assets/thumbnail.jpg');
  background-size: cover;
  background-position: center center;
`

export default CardImage
