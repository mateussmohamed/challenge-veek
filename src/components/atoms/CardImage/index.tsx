import styled from 'styled-components'

export const CardImage = styled.img`
  width: 100%;
  height: 170px;
  background-color: #25262c;
  background-image: url('/assets/thumbnail.jpg');
  background-size: cover;
  background-position: center center;
`
