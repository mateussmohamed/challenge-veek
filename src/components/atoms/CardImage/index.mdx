---
name: CardImage
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import CardImage from './index'

# CardImage
<PropsTable of={CardImage} />

<Playground>
  <CardImage />
</Playground>
