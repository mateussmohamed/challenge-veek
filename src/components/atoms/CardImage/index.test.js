import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardImage from './index'

describe('[<CardImage />]', () => {
  const wrap = (props = {}) => shallow(<CardImage {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap()

    expect(wrapper).toMatchSnapshot()
  })
})
