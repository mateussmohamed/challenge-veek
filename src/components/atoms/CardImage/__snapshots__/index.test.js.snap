// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<CardImage />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "CardImage-sc-12t4rh8-0",
        "isStatic": true,
        "rules": Array [
          "width:100%;height:170px;background-color:#25262c;background-image:url('/assets/thumbnail.jpg');background-size:cover;background-position:center center;",
        ],
      },
      "displayName": "CardImage",
      "render": [Function],
      "styledComponentId": "CardImage-sc-12t4rh8-0",
      "target": "img",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
